app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt

```diff
@@ -109,12 +119,18 @@ fun BusScheduleApp(
                 route = BusScheduleScreens.RouteSchedule.name + "/{$busRouteArgument}",
                 arguments = listOf(navArgument(busRouteArgument) { type = NavType.StringType })
             ) { backStackEntry ->
+                val layoutDirection = LocalLayoutDirection.current
                 val stopName = backStackEntry.arguments?.getString(busRouteArgument)
                     ?: error("busRouteArgument cannot be null")
                 val routeSchedule by viewModel.getScheduleFor(stopName).collectAsState(emptyList())
                 RouteScheduleScreen(
                     stopName = stopName,
+                    modifier = Modifier.padding(
+                        start = innerPadding.calculateStartPadding(layoutDirection),
+                        end = innerPadding.calculateEndPadding(layoutDirection),
+                    ),
                     busSchedules = routeSchedule,
+                    contentPadding = innerPadding,
                     onBack = { onBackHandler() }
```

Comment:
this is bit confusing. 
We either use the content padding as a `Modifier.padding` or as `contentPadding` not both.